app/controllers/admin/attachments_controller.rb

```diff
@@ -6,7 +6,9 @@ class Admin::AttachmentsController < Admin::BaseController
 
   rescue_from Mysql2::Error, with: :handle_duplicate_key_errors_caused_by_double_create_requests
 
-  def index; end
+  def index
+    render_design_system("index", "index_legacy", next_release: true)
```

Comment:
I believe that this the index page is release 1.4 so should not be included in the next release yet